File: src/template/pedro/Suma.java
package template.pedro;

class Suma extends Operacion {
    Suma() {
    }

    Suma(int redondeo) {
        this.redondeo = redondeo;
    }

    void getOperacion() {
        System.out.println("operación: suma");
    }

    void calcula() {
        resultado = primerValor + segundoValor;
    }
}

File: manuel/template/Suma.php
<?php 
	require_once "Operacion.php";
	
	class Suma extends Operacion{
		public function calcula()
		{
			$valor1 = $this->valor1;
			$valor2 = $this->valor2;
			$this->resultado = $valor1 - $valor2;
			// return $this->valor1 - $this->valor2;
		}

		public function getOperacion()
		{
			echo 'Suma <br/>';
		}
	}
?>